Tests are disable and failing in master

when your github action runs:
https://github.com/spotify/backstage/runs/552952974?check_suite_focus=true

The tests don't run in master because you have --since origin/master
```
Run yarn test
yarn run v1.22.1
$ cross-env CI=true lerna run test --since origin/master -- --coverage
lerna notice cli v3.20.2
lerna info ci enabled
lerna notice filter changed since "origin/master"
lerna info Looking for changed packages since origin/master
lerna success run No packages found with the lifecycle script 'test'
Done in 0.58s.
```

If I remove --since origin/master to run all tests,  the tests are failing with:
```
lerna ERR! yarn run test --coverage stderr:
 FAIL  src/components/WelcomePage/ToggleFeatureFlagButton.test.tsx
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    src/components/WelcomePage/ToggleFeatureFlagButton.test.tsx:25:8 - error TS2307: Cannot find module '@backstage/core'.

    25 } from '@backstage/core';
              ~~~~~~~~~~~~~~~~~

 FAIL  src/plugin.test.ts
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    src/plugin.ts:17:30 - error TS2307: Cannot find module '@backstage/core'.

    17 import { createPlugin } from '@backstage/core';
                                    ~~~~~~~~~~~~~~~~~
    src/plugin.ts:22:14 - error TS7031: Binding element 'router' implicitly has an 'any' type.

    22   register({ router, featureFlags }) {
                    ~~~~~~
    src/plugin.ts:22:22 - error TS7031: Binding element 'featureFlags' implicitly has an 'any' type.

    22   register({ router, featureFlags }) {
                            ~~~~~~~~~~~~

 FAIL  src/components/WelcomePage/WelcomePage.test.tsx
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    src/components/WelcomePage/WelcomePage.test.tsx:26:8 - error TS2307: Cannot find module '@backstage/core'.

    26 } from '@backstage/core';
              ~~~~~~~~~~~~~~~~~

 FAIL  src/components/WelcomePage/ErrorButton.test.tsx
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    src/components/WelcomePage/ErrorButton.test.tsx:20:55 - error TS2307: Cannot find module '@backstage/core'.

    20 import { ApiRegistry, errorApiRef, ApiProvider } from '@backstage/core';
                                                             ~~~~~~~~~~~~~~~~~

Failed to collect coverage from /Users/557659/backstage/plugins/welcome/src/plugin.ts
ERROR: TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
src/plugin.ts:17:30 - error TS2307: Cannot find module '@backstage/core'.

17 import { createPlugin } from '@backstage/core';
                                ~~~~~~~~~~~~~~~~~
src/plugin.ts:22:14 - error TS7031: Binding element 'router' implicitly has an 'any' type.

22   register({ router, featureFlags }) {
                ~~~~~~
src/plugin.ts:22:22 - error TS7031: Binding element 'featureFlags' implicitly has an 'any' type.

22   register({ router, featureFlags }) {
                        ~~~~~~~~~~~~
STACK: 
Failed to collect coverage from /Users/557659/backstage/plugins/welcome/src/components/Timer/Timer.tsx
ERROR: TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
src/components/Timer/Timer.tsx:18:29 - error TS2307: Cannot find module '@backstage/core'.

18 import { HeaderLabel } from '@backstage/core';
                               ~~~~~~~~~~~~~~~~~
STACK: 
Failed to collect coverage from /Users/557659/backstage/plugins/welcome/src/components/WelcomePage/ToggleFeatureFlagButton.tsx
ERROR: TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
src/components/WelcomePage/ToggleFeatureFlagButton.tsx:19:62 - error TS2307: Cannot find module '@backstage/core'.

19 import { FeatureFlagState, featureFlagsApiRef, useApi } from '@backstage/core';
                                                                ~~~~~~~~~~~~~~~~~
STACK: 
Failed to collect coverage from /Users/557659/backstage/plugins/welcome/src/components/WelcomePage/ErrorButton.tsx
ERROR: TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
src/components/WelcomePage/ErrorButton.tsx:19:37 - error TS2307: Cannot find module '@backstage/core'.

19 import { errorApiRef, useApi } from '@backstage/core';
                                       ~~~~~~~~~~~~~~~~~
STACK: 
Failed to collect coverage from /Users/557659/backstage/plugins/welcome/src/components/WelcomePage/WelcomePage.tsx
ERROR: TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
src/components/WelcomePage/WelcomePage.tsx:36:8 - error TS2307: Cannot find module '@backstage/core'.

36 } from '@backstage/core';
          ~~~~~~~~~~~~~~~~~
STACK: 
Test Suites: 4 failed, 4 total
Tests:       0 total
Snapshots:   0 total
Time:        39.516s
Ran all test suites.
Command 'web-scripts' exited with code 1
error Command failed with exit code 1.

lerna ERR! yarn run test --coverage exited 1 in '@backstage/plugin-welcome'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/.github/workflows/frontend.yml
+++ b/.github/workflows/frontend.yml
@@ -48,12 +48,21 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+
       - name: yarn install
         run: yarn install --frozen-lockfile
-      - run: yarn lint
-      - run: yarn build
-      - run: yarn test
+
+      - name: lint
+        run: yarn lerna -- run lint --since origin/master
+
+      - name: build
+        run: yarn build
+
+      - name: test
+        run: yarn lerna -- run test --since origin/master -- --coverage
+
       - name: yarn bundle, if app was changed
-        run: git diff --quiet origin/master HEAD -- packages/app packages/core || yarn bundle
+        run: git diff --quiet origin/master HEAD -- yarn.lock packages/app packages/core || yarn bundle
+
       - name: verify storybook
         run: yarn workspace storybook build-storybook

--- a/.github/workflows/master.yml
+++ b/.github/workflows/master.yml
@@ -47,16 +47,25 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           registry-url: https://registry.npmjs.org/ # Needed for auth
+
       - name: yarn install
         run: yarn install --frozen-lockfile
-      - run: yarn lint
-      - run: yarn build
-      - run: yarn test
+
+      - name: lint
+        run: yarn lerna -- run lint
+
+      - name: build
+        run: yarn build
+
+      - name: test
+        run: yarn lerna -- run test -- --coverage
+
       # Publishes current version of packages that are not already present in the registry
       - name: publish
-        run: npx --no-install lerna publish from-package --yes
+        run: yarn lerna -- publish from-package --yes
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
       # Tags the commit with the version in the core package if the tag doesn't exist
       - uses: Klemensas/action-autotag@1.2.3
         with:

--- a/package.json
+++ b/package.json
@@ -12,6 +12,7 @@
     "create-plugin": "backstage-cli create-plugin",
     "release": "if [ \"$(git symbolic-ref --short HEAD)\" = master ]; then echo \"don't try to release master\"; exit 1; else lerna version --no-push; fi",
     "lint": "cross-env CI=true lerna run lint --since origin/master --",
+    "lerna": "lerna",
     "storybook": "yarn workspace storybook start"
   },
   "workspaces": {

--- a/package.json
+++ b/package.json
@@ -41,8 +41,8 @@
     }
   },
   "lint-staged": {
-      "*.{js,jsx,ts,tsx}": [
-        "prettier --config ./prettier.config.js  --list-different --write"
-      ]
+    "*.{js,jsx,ts,tsx}": [
+      "prettier --config ./prettier.config.js  --list-different --write"
+    ]
   }
 }

--- a/package.json
+++ b/package.json
@@ -8,10 +8,10 @@
     "start": "yarn build && yarn workspace example-app start",
     "bundle": "yarn build && yarn workspace example-app bundle",
     "build": "lerna run build",
-    "test": "cross-env CI=true lerna run test --since origin/master -- --coverage",
+    "test": "yarn build && lerna run test --since origin/master -- --coverage",
+    "lint": "lerna run lint --since origin/master --",
     "create-plugin": "backstage-cli create-plugin",
     "release": "if [ \"$(git symbolic-ref --short HEAD)\" = master ]; then echo \"don't try to release master\"; exit 1; else lerna version --no-push; fi",
-    "lint": "cross-env CI=true lerna run lint --since origin/master --",
     "lerna": "lerna",
     "storybook": "yarn workspace storybook start"
   },

